src/mantarray_desktop_app/main_process/process_monitor.py

```diff
@@ -225,7 +225,10 @@ def _check_and_handle_server_to_main_queue(self) -> None:
             else:
                 raise NotImplementedError(f"Unrecognized shutdown command from Server: {command}")
         elif communication_type == "update_user_settings":
-            new_values = communication["content"]
+            # if content does not exist then check config_settings
```

Comment:
Are you adding this for the test below? if so, why not just keep it content and change the test?